acai-ios/BowlOrderDetailViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  BowlOrderDetailViewController.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+
+protocol DidSelectOption: class {
```

Comment:
you should name protocols w/ `Delegate` at the end so this should be `DidSelectOptionDelegate`